src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java

```diff
@@ -29,6 +31,11 @@
 @QuarkusTestResource(TestLifecycleManager.class)
 public class EndpointServiceTest {
 
+    @BeforeEach
+    void beforeEach() {
+        RestAssured.basePath = "/api/integrations/v1.0";
```

Comment:
This ``/api/integrations/v1.0`` string is in so many places, how will the refactoring work if we end up with 1.1?

Reply:
Yeah, wanted to turn that into constants, but forgot.